qoqo-qryd/tests/integration/tweezer_devices.rs

```diff
@@ -527,7 +527,7 @@ fn test_number_qubits() {
     // Setup fake preconfigured device
     let mut exp = TweezerDevice::new(None, None, None);
     exp.add_layout("default").unwrap();
-    exp.current_layout = Some("default".to_string());
+    exp.switch_layout("default").unwrap();
```

Comment:
Is this testing an earlier changed feature?

Reply:
I change it to this so that the Rust part and the Pyo3 one follow the exact steps.
In reality, there's no real difference as switching to a newly created layout (so without setting any Tweezer information) does not modify the qubit->tweezer information (by default, switching to a layout that contains Tweezer info automatically fills the mapping; this is tested elsewhere).
So, just for simmetry reasons.